<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="CORPLPW100314" failures="2" tests="8" name="Test" time="172.84" errors="0" timestamp="2021-04-20T18:20:04 IST">
  <testcase classname="com.athmatest.qa.testcases.SampleTest" name="myFirstTest" time="0.038"/>
  <testcase classname="com.athmatest.qa.testcases.SampleTest" name="mySecondTest" time="6.438"/>
  <testcase classname="com.athmatest.qa.testcases.SampleTest" name="myThirdTest" time="12.517">
    <skipped/>
  </testcase> <!-- myThirdTest -->
  <testcase classname="com.athmatest.qa.testcases.SampleTest" name="myThirdTest" time="8.323">
    <skipped/>
  </testcase> <!-- myThirdTest -->
  <testcase classname="com.athmatest.qa.testcases.SampleTest" name="myThirdTest" time="7.142">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.athmatest.qa.testcases.SampleTest.myThirdTest(SampleTest.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- myThirdTest -->
  <testcase classname="com.athmatest.qa.testcases.SampleTest" name="myFourthTest" time="7.48">
    <skipped/>
  </testcase> <!-- myFourthTest -->
  <testcase classname="com.athmatest.qa.testcases.SampleTest" name="myFourthTest" time="7.317">
    <skipped/>
  </testcase> <!-- myFourthTest -->
  <testcase classname="com.athmatest.qa.testcases.SampleTest" name="myFourthTest" time="7.071">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.athmatest.qa.testcases.SampleTest.myFourthTest(SampleTest.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- myFourthTest -->
</testsuite> <!-- Test -->
